app/views/components/_email-signup-section.html.erb

```diff
@@ -0,0 +1,30 @@
+<%
+  local_assigns[:heading_level] ||= 3
+  local_assigns[:margin_bottom] ||= 0
+  link_text ||= false
+  link_href ||= false
+  intro ||= false
+  background ||= false
+  classes = []
+  data ||= false
+
+  shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new(local_assigns)
+
+  classes << shared_helper.get_margin_bottom
+  classes << "app-c-email-signup--no-intro" unless intro
+  classes << "app-c-email-signup--with-background-and-border" if background
+%>
+<% if link_text && link_href %>
+  <div class="app-c-email-signup dont-print <%= classes.join(' ') %>">
```

Comment:
Apparently you can use `govuk-!-display-none-print` instead of `dont-print` (which might disappear at some point).

Reply:
✔️